pkg/rancher-desktop/assets/scripts/install-k3s

```diff
@@ -30,7 +30,16 @@ fi
 # Add images
 IMAGES="/var/lib/rancher/k3s/agent/images"
 mkdir -p "${IMAGES}"
-ln -s -f "${K3S_DIR}/k3s-airgap-images-${ARCH}.tar" "${IMAGES}"
+IMAGEPATH="${K3S_DIR}/k3s-airgap-images-${ARCH}"
+if [ -f "${IMAGEPATH}.tar.zst" ]; then
+    ln -s -f "${IMAGEPATH}.tar.zst" "${IMAGES}"
+fi
+if [ -f "${IMAGEPATH}.tar.gz" ]; then
+    ln -s -f "${IMAGEPATH}.tar.gz" "${IMAGES}"
+fi
```

Comment:
I think we should remove the code supporting `*.tar.gz` files because it is never used. We don't download them because whenever they exist, there will also be a corresponding `*.tar.zst` file, as both compression methods were added to the k3s release process at the same time: https://github.com/k3s-io/k3s/pull/3169.

Having dead code that is not being tested either just adds to tech debt; it makes it harder to understand and modify the code in the future. I understand that it is just a few lines in this case, but you have to fight this from the beginning! 😄